node/resolvers/Mutations/Users.ts

```diff
@@ -11,6 +12,36 @@ import {
 
 const config: any = currentSchema('b2b_users')
 
+const setChangeSession = async (sessionParameters: any, countRetry = 0) => {
+  const retry = 5
+
+  const {
+    context: {
+      clients: { session },
+      vtex: { logger },
+    },
+    publicKey,
+    value,
+    sessionCookie,
+  } = sessionParameters
+
+  try {
+    await session.updateSession(publicKey, value, [], sessionCookie)
+  } catch (error) {
+    logger.error({
+      error,
+      message: 'setChangeSession.error',
+    })
+    if (countRetry < retry) {
+      countRetry++
+      logger.info({ message: `setChangeSession RETRY ${countRetry}` })
+      setTimeout(() => {
+        setChangeSession(sessionParameters, countRetry)
+      }, 1000)
```

Comment:
can we move the timeout to a constant? Do we really need 1s? Could it be less? Maybe 500ms?